task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-validation
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+  steps:
+    - name: check-labels-and-binaries
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: /work
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: /work
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
+
+        mkdir confdir
+        if ! oc image extract $(params.IMAGE_URL) --file /bin/opm --file /bin/grpc_health_probe --path configs/*:confdir/ ; then
+        echo "Unable to extract image! Skipping checking binaries!"
+          HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
+          '{result: "FAILURE", timestamp: $date, successes: 0, failures: 1}')
```

Comment:
We can set the result to `ERROR` here to signify the error with extracting the image.

Reply:
Set.